.github/workflows/changelog-check.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - master
```

Comment:
I think once it will merged to paster.. it will fail the changelog there if we leave it as is.

Reply:
that shouldn't happen since we want it to run in master branch though...

Reply:
If the PR is green, master should be green.. it will be too late if the error is on master :)